graal-tools/src/main/scala/io/cloudstate/graaltools/AkkaActorRegisterFeature.scala

```diff
@@ -1,30 +1,25 @@
 package io.cloudstate.graaltools
 
 import com.oracle.svm.core.annotate.AutomaticFeature
-import org.graalvm.nativeimage.hosted.Feature.QueryReachabilityAccess
 import org.graalvm.nativeimage.hosted.{Feature, RuntimeReflection}
 
-import scala.collection.JavaConverters._
-import java.util.concurrent.ConcurrentHashMap
-
 @AutomaticFeature
 final class AkkaActorRegisterFeature extends Feature {
-  private[this] final val cache = {
-    val c = ConcurrentHashMap.newKeySet[String]
+  private[this] final val ignoreClass = Set(
     // FIXME the following are configured in akka-graal-config, remove this when migrating from it
-    c.add("akka.io.TcpConnection")
-    c.add("akka.io.TcpListener")
-    c.add("akka.stream.impl.fusing.ActorGraphInterpreter")
-    c
-  }
+    "akka.io.TcpConnection",
+    "akka.io.TcpListener",
+    "akka.stream.impl.fusing.ActorGraphInterpreter"
+  ).contains(_)
```

Comment:
Set.apply calls contains, so not needed?

```suggestion
  )
```

Reply:
Yeah, it does, I prefer to explicitly make it a predicate (it's this whole "intentional vs extension" set thing, which I won't go into).  Oh, and this way was terser than type annotating it as a function.  Anyways, happy to drop it.